nexus/reconfigurator/planning/src/example.rs

```diff
@@ -146,29 +235,15 @@ impl ExampleSystem {
                 .unwrap();
         }
 
-        ExampleSystem {
+        // Previously ExampleSystem itself used to carry around the blueprint,
```

Comment:
I don't think we should be referring to what prior versions of the code did in comments. It adds extra cognitive overhead for the reader. John left a similar comment on a PR of mine recently after a chat with Dave regarding comments. 

However, the rest of the comment is useful, so how about something like:

> The blueprint evolves separately from the system -- so it's returned as a separate value.

Reply:
I personally like short notes about battles won in the comments (not just the commit history), but I don't care too much so I'll change it :)